llpc/lower/llpcSpirvLowerRayQuery.cpp

```diff
@@ -594,6 +594,55 @@ Value *SpirvLowerRayQuery::getDispatchId() {
   return dispatchId;
 }
 
+// =====================================================================================================================
+// Create instructions to get BVH SRD given the expansion and box sort mode at the current insert point
+//
+// @param expansion : Box expansion
+// @param boxSortMode : Box sort mode
+Value *SpirvLowerRayQuery::createGetBvhSrd(llvm::Value *expansion, llvm::Value *boxSortMode) {
+  const auto *rtState = m_context->getPipelineContext()->getRayTracingState();
+  assert(rtState->bvhResDesc.dataSizeInDwords == 4);
+
+  // Construct image descriptor from rtstate.
+  Value *bvhSrd = UndefValue::get(FixedVectorType::get(m_builder->getInt32Ty(), 4));
```

Comment:
Use PoisonValue while you're at it, since LLVM wants to long-term transition away from `undef` and towards `poison`.